src/main/java/seedu/address/model/person/Person.java

```diff
@@ -26,21 +27,25 @@
     private final Address address;
     private final Remark remark;
     private final Set<Tag> tags = new HashSet<>();
-    private ArrayList<Group> groups = new ArrayList<Group>();
```

Comment:
Refer to comment above about the list of group

Reply:
Usually we don't create separate ArrayList for the groups for an individual class, we use one for all the classes.

Reply:
That won't work for this case as mentioned in the comment above. The GroupList class that we used is for a different purpose which is to track and manage all the group created. What we need is a list of group for the specific person. Based on your current code, 1 person is only allowed to be in 1 group. I think it should be that 1 person could be in any number of groups. It is either we need a new class, or use arraylist, list, set, etc. to store it,